Broadcom/symantec-endpoint-protection/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,52 @@
+[{
+    "value": "Allowed execution of {file.path} on {host.hostname} ({host.ip})",
+    "conditions": [
+        {
+            "field": "event.reason",
+            "value": "SONAR detection now allowed"
+        },
+        {
+            "field": "host.hostname"
+        },
+        {
+            "field": "host.ip"
+        },
+        {
+            "field": "file.path"
+        }
+    ],
+    "relationships": [
+        {
+            "source": "host.hostname",
+            "target": "file.path",
+            "type": "execute"
+        }
+    ]
+},{
+    "value": "Virus found on {host.hostname} ({host.ip}): {file.path} was {event.action}",
```

Comment:
I didn't get the end of the sentence `was {event.action}`.

I suggest that you modified this smart description by using the following sentence

```
    "broadcom": {
      "endpoint_protection": {
        "source": "Auto-Protect scan",
```

**identified**
```
    "file": {
      "path": "C:\\Users\\admin\\Desktop\\test.txt",
```

**as**
```
    "rule": {
      "name": "Virus",
```

**and applied the action**

```
  "broadcom": {
        "action": {
          "main": "Cleaned by deletion",
```